sss/report.py

```diff
@@ -0,0 +1,86 @@
+import glob
+import os
+from datetime import datetime, date
+
+import numpy as np
+import pandas as pd
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Float,
+    Boolean,
+    Date)
+
+from . import preprocess 
+from .base import Base,DB, DeclarativeDB
+from .base import db_url as default_db_url
+#from sss import REPORT
+
+# declare REPORT data columns and data type
+class REPORT(Base):
+    '''
+    Attributes
+    ----------
+    year: int
+        year of report
+    state: str
+        name of state
+    analysis_type: str
+        analysis type of sss, e.g. full, partial
+    cpi_month: str
+        cpi month of the report
+    cpi_year
+        cpi year of the report
+    update_date: date
+        update date
+    update_person: str
+        who update the report
+    '''
+    __tablename__ = 'report'
+    year = Column('year', Integer, primary_key = True)
+    state = Column('state', String, primary_key = True)
+    analysis_type = Column('analysis_type', String, primary_key = True)
+    cpi_month = Column('cpi_month', String)
+    cpi_year =  Column('cpi_year', Integer)
+    update_date = Column('update_date', Date)
+    update_person = Column('update_person', String)
+
+def add_report(path):
+    """
+    Reads report data into data frame and perprare for database report table
+    Parameters
```

Comment:
There should be an empty line between the description and the Parameters section. And another one between  Parameters and Returns